database/repository/candle/candle_test.go

```diff
@@ -75,60 +77,36 @@ func TestInsert(t *testing.T) {
 			},
 			seedDB: seedDB,
 		},
-	}
-
-	for x := range testCases {
-		test := testCases[x]
-
-		t.Run(test.name, func(t *testing.T) {
-			if !testhelpers.CheckValidConfig(&test.config.ConnectionDetails) {
+	} {
+		t.Run(tc.name, func(t *testing.T) {
+			if !testhelpers.CheckValidConfig(&tc.config.ConnectionDetails) {
 				t.Skip("database not configured skipping test")
 			}
 
-			dbConn, err := testhelpers.ConnectToDatabase(test.config)
-			if err != nil {
-				t.Fatal(err)
-			}
+			dbConn, err := testhelpers.ConnectToDatabase(tc.config)
+			require.NoError(t, err)
 
-			if test.seedDB != nil {
-				err = test.seedDB(false)
-				if err != nil {
-					t.Fatal(err)
-				}
+			if tc.seedDB != nil {
+				require.NoError(t, tc.seedDB(false))
 			}
 
 			data, err := genOHCLVData()
-			if err != nil {
-				t.Fatal(err)
-			}
-
+			require.NoError(t, err)
 			r, err := Insert(&data)
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.NoError(t, err)
 
-			if r != 365 {
-				t.Errorf("unexpected number inserted: %v", r)
-			}
+			assert.Equal(t, uint64(365), r)
 
 			d, err := DeleteCandles(&data)
-			if err != nil {
-				t.Fatal(err)
-			}
-			if d != 365 {
-				t.Errorf("unexpected number deleted: %v", d)
-			}
-
-			err = testhelpers.CloseDatabase(dbConn)
-			if err != nil {
-				t.Error(err)
-			}
+			require.NoError(t, err)
+			assert.Equal(t, int64(365), d)
```

Comment:
uint64?

Reply:
sqlboiler returns a int64 from the DB side so it's int64's all the way down, even though I'm not sure how you could ever delete negative records unless you overflow it 🤷